db/migrate/20200624104324_devise_create_users.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+class DeviseCreateUsers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :users do |t|
```

Comment:
To prevent database enumeration, the tables should rather be using UUID's for all id's https://itnext.io/using-uuids-to-your-rails-6-application-6438f4eeafdf